synapse/lib/types.py

```diff
@@ -1601,7 +1602,7 @@ def postTypeInit(self):
 
     def norm(self, valu):
         try:
-            s_common.reqjsonsafe(valu)
+            s_json.reqjsonsafe(valu)
```

Comment:
Adding a note here that we need to discuss if any data at rest would now not pass re-normalization and how to handle it